bottlecap/src/traces/trace_agent.rs

```diff
@@ -393,6 +393,24 @@ impl TraceAgent {
 
         for chunk in &mut traces {
             for span in chunk.iter_mut() {
+                // If the aws.lambda.load span is found, we're in Python or Node.
+                // We need to update the trace ID of the cold start span, send the cold start span,
+                // and reparent the aws.lambda.load span to the cold start span.
+                if span.name == "aws.lambda.load" {
+                    let mut invocation_processor = invocation_processor.lock().await;
+                    let cold_start_span_id = invocation_processor
+                        .send_cold_start_span(
+                            &tags_provider,
+                            &trace_processor,
+                            &trace_tx,
+                            span.trace_id,
+                        )
+                        .await;
+                    if let Some(cold_start_span_id) = cold_start_span_id {
+                        span.parent_id = cold_start_span_id;
+                    }
```

Comment:
Is there a way to not trigger a send here and instead just get the `span_id` while modifying the cold start on the other end?

I think we could separate this, as opposed to do all things at once, wdyt?